lib/environment.js

```diff
@@ -21,6 +21,9 @@ exports.Environment = Environment = function(options) {
       filters: options.filters,
       tags: options.tags
    };
+   if (options.i18n) {
```

Comment:
If you remove this `if` and its body, then the i18n configuration is still available at `env.config.i18n`. Tags should put their configuration there and not directly at the environment.